### PR TITLE
DOC: glossary for minimum-norm estimate

### PR DESCRIPTION
**Pull request by klankinen**
Added "minimum-norm estimate to the glossary"

@drammock

### CODE REVIEW

File: doc/glossary.rst [hunk 1]
```diff
@@ -137,6 +137,14 @@ general neuroimaging concepts. If you think a term is missing, please consider
     label
         A :class:`Label` refers to a region in the cortex, also often called
         a region of interest (ROI) in the literature.
+        
+    minimum-norm estimation
+        Minimum-norm estimation (abbr. ''MNE'') can be used to generate a distributed 
+        map of activation on a :term:'source space', usually on a cortical surface.
```
Comment (larsoner) [inline doc/glossary.rst]: probably need backticks rather than single quotes on these to get it to render properly in Sphinx

**Comment by drammock** [comment]
Looks good!  Here is the rendering: https://15787-1301584-gh.circle-artifacts.com/0/dev/glossary.html#term-minimum-norm-estimation

**Comment by drammock** [comment]
Merging without waiting for Travis / Azure since this only touches the docs pages.  Thanks @klankinen!

**Comment by agramfort** [comment]
Thanks @klankinen <https://github.com/klankinen>!

>

**Pull request merged**